src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private final int side;
+
+    public Square(String color,int side) {
+        super(color);
+        this.side = side;
+    }
+
+    @Override
+    public double area() {
```

Comment:
The method name 'area' should be a verb, consider renaming it to something like 'calculateArea' to better reflect the action it performs.